meinberlin/apps/notifications/api.py

```diff
@@ -59,6 +59,7 @@ def interactions(self, request):
         where target object can be a proposal, an idea, a comment.
         """
         base_qs = self.get_queryset().filter(action__target_creator=request.user)
+        unread_count = base_qs.filter(read=False).count()
```

Comment:
I think this would count every rating event as a single notification, but further down, we aggregate them into one notification. It might be a design decision, but maybe it's confusing when you _see_ 3 notifications but it says you have 6 unread because three of them are aggregated ratings you got for one of your comment.